CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.5)
```

Comment:
Why 3.5? Cmake-3.1 already has `CMAKE_CXX_STANDARD` - might be a bit more conservative.